Target-MVD-Mathias/ViewControllers/SignUpViewController.swift

```diff
@@ -200,29 +204,17 @@ extension SignUpViewController: UITextFieldDelegate {
   func textFieldDidEndEditing(_ textField: UITextField) {
     switch textField {
       case nameTextField:
-        if nameIsInvalid() {
-          showErrorInForm(textField: nameTextField, errorLabel: nameErrorLabel)
-        } else {
-          hideErrorInForm(textField: nameTextField, errorLabel: nameErrorLabel)
-        }
+        toggleErrorInForm(error: nameIsInvalid(), textField: nameTextField, errorLabel: nameErrorLabel)
       case emailTextField:
-        if emailIsInvalid() {
-          showErrorInForm(textField: emailTextField, errorLabel: emailErrorLabel)
-        } else {
-          hideErrorInForm(textField: emailTextField, errorLabel: emailErrorLabel)
-        }
+        toggleErrorInForm(error: emailIsInvalid(), textField: emailTextField, errorLabel: emailErrorLabel)
       case passwordTextField, confirmPasswordTextField:
         checkPasswordErrors()
         
         if textField == passwordTextField {
```

Comment:
After reviewing the code I realize it's not the same, because I just want it to change when the user edits the password field, not when the confirm password field is edited